### ISSUE TITLE
Capitalization typo in Explicit MainActor Context

### ISSUE THREAD

**Issue by bjo0** [apple/swift-migration-guide#122]
https://www.swift.org/migration/documentation/swift-6-concurrency-migration-guide/incrementaladoption/#Explicit-MainActor-Context

```
if the assumption was incorrect. that would otherwise be invisible to the compiler.
                                 ^
```

**Comment by mattmassicotte** [comment]
Ohh, thanks for finding this!

**Comment by mattmassicotte** [comment]
Fixed with #125
